### PR TITLE
add MultiSystem for ASE traj(or any other format can be read by ASE)

### PR DESCRIPTION
**Pull request by bigd4**
add MultiSystem for ASE traj(or any other format can be read by ASE)
fix bug(?) for ASE 'to_labeled_system'

### CODE REVIEW

File: dpdata/plugins/ase.py [hunk 1]
```diff
@@ -1,8 +1,46 @@
 from dpdata.format import Format
+import numpy as np
+try:
+    import ase.io
+except:
+    raise Exception('ASE module not installed')
```
Comment (njzjz) [inline dpdata/plugins/ase.py:6]: This line should be removed, as `ase` is only an optional package.

File: dpdata/plugins/ase.py [hunk 2]
```diff
@@ -1,8 +1,46 @@
 from dpdata.format import Format
+import numpy as np
+try:
+    import ase.io
+except:
```
Comment (njzjz) [inline dpdata/plugins/ase.py:5]: [Do not use base `except:` clause.](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)

File: dpdata/plugins/ase.py [hunk 3]
```diff
@@ -1,8 +1,46 @@
...
+            atom_types = np.array([atom_names.index(symbol) for symbol in symbols]).astype(int)
+    
+            cells = np.array([atoms.cell[:]]).astype('float32')
+            coords = np.array([atoms.get_positions()]).astype('float32')
+            energies = np.array([atoms.get_potential_energy(force_consistent=True)]).astype('float32')
+            forces = np.array([atoms.get_forces()]).astype('float32')
+            try:
+                stress = atoms.get_stress(False)
+                virials = np.array([-atoms.get_volume() * stress]).astype('float32')
+            except:
```
Comment (njzjz) [inline dpdata/plugins/ase.py:29]: Do not use base `except:` clause.

File: dpdata/plugins/ase.py [hunk 4]
```diff
@@ -1,8 +1,46 @@
...
+except:
+    raise Exception('ASE module not installed')
 
 
 @Format.register("ase/structure")
 class ASEStructureFormat(Format):
+    def from_labeled_system(self, data, **kwargs):
+        return data
+
+    def from_multi_systems(self, file_name, begin=None, end=None, step=None, fmt='traj', **kwargs):
```
Comment (njzjz) [inline dpdata/plugins/ase.py:14]: Please consider add unit tests for this new method.

**Review by njzjz — CHANGES_REQUESTED** [review]
Please set the target branch to `devel` instead of `master` (you can click the `edit` button).

**Review by njzjz — APPROVED** [review]

**Review by amcadmus — APPROVED** [review]

**Pull request merged**